istioctl/pkg/waypoint/waypoint.go

```diff
@@ -43,13 +50,95 @@ import (
 )
 
 var (
-	revision = ""
-
+	revision      = ""
+	waitReady     bool
+	waitTimeout   time.Duration
 	allNamespaces bool
+	revisionLabel = "istio.io/rev"
```

Comment:
nit: This can be `label.IoIstioRev.Name`.

Reply:
After switching to checking on the gateway API, the variable for the revisionLabel has now been removed